SETUP.md

```diff
@@ -17,8 +17,8 @@ to pass again.  The test file is located in the `tests` directory.   You can
 also remove the ignore flag from all the tests to get them to run all at once
 if you wish.
 
-Make sure to read the [Crates and Modules](crates-and-modules) chapter if you
+Make sure to read the [Crates and Modules](https://doc.rust-lang.org/stable/book/crates-and-modules.html) chapter if you
```

Comment:
Ah, I see what happened here. They're trying to use a reference style link, which is two sets of square brackets:

```
[the anchor text][link]

[link]: http://google.com
```

[the anchor text][link]

[link]: http://google.com